server/src/main/java/net/cryptic_game/backend/server/server/daemon/DaemonServerCodec.java

```diff
@@ -25,14 +25,14 @@ public DaemonServerCodec(final DaemonHandler daemonHandler, final DaemonEndpoint
     }
 
     private void handleResponse(final JsonObject jsonObject, final ApiClient client) {
-        UUID tag = JsonUtils.getUUID(jsonObject, "tag");
+        UUID tag = JsonUtils.fromJson(jsonObject.get("tag"), UUID.class);
         if (this.daemonHandler.isRequstOpen(tag))
             this.daemonHandler.respondToClient(jsonObject);
         else
             client.get(Daemon.class).getChannel()
-                    .writeAndFlush(JsonBuilder.anJSON()
-                            .add("tag", tag)
-                            .add("info", JsonBuilder.anJSON(ApiResponseType.BAD_REQUEST.serialize()).add("message", "TOO_LATE").build())
+                    .writeAndFlush(JsonBuilder
+                            .create("tag", tag)
+                            .add("info", JsonBuilder.create(ApiResponseType.BAD_REQUEST.serialize()).add("message", "TOO_LATE").build())
```

Comment:
you can remove `build()`

Reply:
you dont need `serialize()`

Reply:
> 
> 
> you dont need `serialize()`

I do need `serialize()` as `JsonBuilder.create()` requires a `JsonObject` not an `ApiResponseType`.